staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -513,7 +517,7 @@ function kube::codegen::gen_client() {
                 plural_exceptions="$2"
                 shift 2
                 ;;
-            "--prefers-protobuf")
+              "--prefers-protobuf")
```

Comment:
Wrong whitespace change

Reply:
Fixed in 4f4bedd0888034127e6d309ae3ede2e1e2cae9c5